File: examples/CBuilder/Compound/ModelEvolution/Common/dSystemTypeInfo.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "dSystemTypeInfo.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "BoldHandles"
#pragma link "BoldSubscription"
#pragma resource "*.dfm"
TdmSystemTypeInfo *dmSystemTypeInfo;
//---------------------------------------------------------------------------
__fastcall TdmSystemTypeInfo::TdmSystemTypeInfo(TComponent* Owner)
  : TDataModule(Owner)
{
}
//---------------------------------------------------------------------------

File: examples/Delphi/Compound/ModelEvolution/Common/dSystemTypeInfo.pas
unit dSystemTypeInfo;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  BoldSubscription, BoldHandles;

type
  TdmSystemTypeInfo = class(TDataModule)
    SystemTypeInfo: TBoldSystemTypeInfoHandle;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  dmSystemTypeInfo: TdmSystemTypeInfo;

implementation

uses dModel, ModelEvClasses;

{$R *.DFM}

end.
